backend/assignments/1/src/main/java/org/example/CSVUtils.java

```diff
@@ -0,0 +1,25 @@
+package org.example;
```

Comment:
why not create a different folder named utils for segregating csv utils, json utils etc...